kube/harbor-scanner-trivy.yaml

```diff
@@ -41,7 +56,22 @@ spec:
             - name: "SCANNER_JOB_QUEUE_REDIS_URL"
               value: "redis://harbor-harbor-redis:6379"
           ports:
-            - containerPort: 8080
+            - name: api-server-port
+              containerPort: 8080
+          livenessProbe:
+            httpGet:
+              path: /probe/healthy
+              port: api-server-port
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            failureThreshold: 3
+          readinessProbe:
+            httpGet:
+              path: /probe/ready
+              port: api-server-port
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            failureThreshold: 3
```

Comment:
It's [good practice](https://srcco.de/posts/kubernetes-liveness-probes-are-dangerous.html) to give the liveness probe a higher failureThreshold than the readiness probe

Reply:
Great post. Thanks for sharing. I changed failure threshold for liveness probe from default 3 to 10.